tests/files/etc/tasks_exit_hook.yaml

```diff
@@ -0,0 +1,174 @@
+setup_hooks:
+  - name: setup_ansible
+    module: script
+    opts:
+      file: /etc/ansible-deploy/hooks/setup_work_dir.sh
+  - name: exit_hook
+    module: script
+    opts:
+      file: /etc/ansible-deploy/hooks/exit_hook.sh
+
+play_items:
```

Comment:
Do we need everything in this file? It's very long which may make understanding of its purpose difficult